src/main/java/org/jabref/gui/journals/ManageJournalAbbreviationsView.java

```diff
@@ -333,7 +336,94 @@ public void commitEdit(String abbreviation) {
             AbbreviationViewModel current = viewModel.currentAbbreviationProperty().get();
             super.commitEdit(abbreviation);
             current.setAbbreviation(oldAbbreviation);
-            viewModel.editAbbreviation(current.getName(), abbreviation);
+            viewModel.editAbbreviation(current.getName(), abbreviation, current.getShortestUniqueAbbreviation());
+        }
+
+        private void createTextField() {
```

Comment:
I think when implementing @Siedlerchr's comment, ths code should be gone, doesn't it?

(A sign that the code is too low-level is that `KeyCode`s are handled)